blockchain/chain.go

```diff
@@ -51,6 +51,13 @@ const (
 	maxSearchDepth = 2880
 )
 
+// voteVersionTuple contains the distiled vote bits and version derived from
```

Comment:
distiled -> distilled

Also the version and bits aren't derived from SSGen, rather they are extracted from a vote.